include/xbt/sysdep.h

```diff
@@ -113,6 +113,8 @@ static XBT_ALWAYS_INLINE void *xbt_realloc(void *p, size_t s) {
     @hideinitializer */
 #define xbt_free(p) free(p) /*nothing specific to do here. A poor valgrind replacement? */
 
+#define xbt_delete(p) (delete[] p) /*nothing specific to do here. A poor valgrind replacement? */
```

Comment:
There is no need for this macro. Write delete[] directly where you've called this macro, instead.